### DISCUSSION TITLE
Mouse support?

### DISCUSSION THREAD

**Discussion by rszyma** [iberianpig/fusuma#299] [Q&A]
Is it possible to configure Fusuma to interpret mouse movements as touchpad gestures, allowing simulating touchpad dragging with a mouse?

**Comment by iberianpig** [comment]
Currently Fusuma cannot interpret external mouse movements as gestures, but it may be possible to implement this by creating a plugin. For your information, fusuma-plugin-thumbsense and fusuma-plugin-remap can map keyboard to mouse buttons and already have the ability to bind to mouse buttons.
